components/brave_wallet_ui/page/screens/backup-wallet/verify-recovery-phrase/verify-recovery-phrase.tsx

```diff
@@ -68,6 +72,7 @@ export const VerifyRecoveryPhrase = () => {
   }, [])
 
   const onSkip = React.useCallback(() => {
+    braveWalletP3A.reportOnboardingAction(OnboardingAction.COMPLETE_RECOVERY_SKIPPED);
     dispatch(WalletPageActions.walletSetupComplete(true))
     history.push(WalletRoutes.OnboardingComplete)
   }, [])
```

Comment:
Should braveWalletP3A be in useCallback's dependencies?

Reply:
i don't think it's necessary because the `ApiProxyContext` shouldn't change during the component lifecycle. however, I will add it just in case i'm wrong.